test/algorithms/set_operations/intersection/alternative_side_strategy.hpp

```diff
@@ -0,0 +1,48 @@
+
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+
+// Copyright (c) 2021 Barend Gehrels, Amsterdam, the Netherlands.
+
+// Licensed under the Boost Software License version 1.0.
+// http://www.boost.org/users/license.html
+
+
+#ifndef BOOST_GEOMETRY_TEST_INTERSECTION_ALTERNATIVE_SIDE_STRATEGY_HPP
+#define BOOST_GEOMETRY_TEST_INTERSECTION_ALTERNATIVE_SIDE_STRATEGY_HPP
+
+#include <boost/geometry/strategies/relate/cartesian.hpp>
+
+
+template <typename SideStrategy>
+class alternate_side_strategy
```

Comment:
Technically this is not side strategy but umbrella strategy using non-standard side strategy.